src/options.rs

```diff
@@ -97,8 +96,11 @@ impl ColumnOptions {
 	}
 
 	pub fn is_valid(&self) -> bool {
-		for size in self.sizes {
-			if size >= crate::table::COMPRESSED_MASK {
+		if self.sizes.len() > crate::table::SIZE_TIERS {
```

Comment:
I was thinking of >= (last entry for the multi entry).

Reply:
good catch